console.error: CustomizeMode:
  Error entering customize mode
  Message: TypeError: messageNode.querySelector(...) is null
  Stack:
    maybeShowTip@resource:///modules/CustomizeMode.jsm:747:7
enter/<@resource:///modules/CustomizeMode.jsm:425:9

--- browser/components/customizableui/CustomizeMode.jsm.orig	2018-01-31 12:45:10 UTC
+++ browser/components/customizableui/CustomizeMode.jsm
@@ -735,14 +735,13 @@ CustomizeMode.prototype = {
       // Put the tip contents in the popup.
       let bundle = this.document.getElementById("bundle_browser");
       const kLabelClass = "customization-tipPanel-link";
-      // eslint-disable-next-line no-unsanitized/property
-      messageNode.innerHTML = bundle.getFormattedString("customizeTips.tip0", [
+      messageNode.unsafeSetInnerHTML(bundle.getFormattedString("customizeTips.tip0", [
         "<label class=\"customization-tipPanel-em\" value=\"" +
           bundle.getString("customizeTips.tip0.hint") + "\"/>",
         this.document.getElementById("bundle_brand").getString("brandShortName"),
         "<label class=\"" + kLabelClass + " text-link\" value=\"" +
         bundle.getString("customizeTips.tip0.learnMore") + "\"/>"
-      ]);
+      ]));
 
       messageNode.querySelector("." + kLabelClass).addEventListener("click", () => {
         let url = Services.urlFormatter.formatURLPref("browser.customizemode.tip0.learnMoreUrl");
